.changeset/wet-buttons-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/wallet-api-core": minor
+---
+
+Support Near family
```

Comment:
Think we can remove this file as seems there are no other files in this folder?

Reply:
@cksanders Not sure what you mean. Can you elaborate on this?